client/components/Balance/index.js

```diff
@@ -0,0 +1,33 @@
+import React, { PropTypes } from 'react';
+
+import formatAmount from 'utils/formatAmount';
+import './style.scss';
+
+const Balance = ({ title, symbol, amount, colorize }) => {
+  const formattedAmount = formatAmount(amount, false);
```

Comment:
ahhh I've been waiting for this 😃 . 
this is kinda business logic, isn't it. If you pre-format the amount in redux OR if you use reselect for obtaining amount, you'd avoid recalculating formatting on every render 😉